### PR TITLE
Add boiler make and model endpoints

### PR DESCRIPTION
**Pull request by LBHCallumM**
## GetBoilerMakes
`GET /api/v2/gasserviceboilers/makes/`

Returns
```javascript
[
  {
    "id": 1,
    "make": ""
  }
]
```

## GetBoilerModels
`GET /api/v2/gasserviceboilers/models/`

Returns
```javascript
[
  {
    "id": 1,
    "model": "",
    "make": {
      "id": 1,
      "make": ""
    }
  }
]
```

## AddBoilerMake
`POST /api/v2/gasserviceboilers/makes/`

Body
```javascript
{
  "make": ""
}
```

Returns
```javascript
{
  "id": 1,
  "make": ""
}
```

## AddBoilerModel
`POST /api/v2/gasserviceboilers/models/`

Body
```javascript
{
  "model": "",
  "makeId": 1
}
```

Returns
```javascript
{
  "id": 1,
  "model": "",
  "make": {
    "id": 1,
    "make": ""
  }
}
```

### CODE REVIEW

**Review by Asura5 — APPROVED** [review]

File: RepairsApi.Tests/V2/Controllers/GasServiceBoilersControllerTests.cs [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+                .Setup(x => x.ExecuteAsync(request))
+                .ReturnsAsync(useCaseResponse);
+
+            // Act
+            var response = await _classUnderTest.AddBoilerModel(request);
+
+            var statusCode = GetStatusCode(response);
+
+            // Assert
+            statusCode.Should().Be((int) HttpStatusCode.BadRequest);
```
[the remark this answers is not in the document]
Comment (LBHCallumM) [reply RepairsApi.Tests/V2/Controllers/GasServiceBoilersControllerTests.cs:162]: I'm not too sure what you mean. Are you suggesting I return a 400 status when the request is invalid, but a 404 when the specified boiler make doesn't exist?

[the remark this answers is not in the document]
Comment (Asura5) [reply RepairsApi.Tests/V2/Controllers/GasServiceBoilersControllerTests.cs:162]: These two cases both return an identical response:
AddBoilerMake_WhenMakeIsEmpty_ReturnsBadRequest
AddBoilerMake_WhenValidRequest_ReturnsCreatedResponse

So, as a user, if I get this response, how do I know what is wrong? Could we include in error message with the response?

[the remark this answers is not in the document]
Comment (Asura5) [reply RepairsApi.Tests/V2/Controllers/GasServiceBoilersControllerTests.cs:162]: Ah - I see that you do in the controller - it's just not in the test case

[the remark this answers is not in the document]
Comment (Asura5) [reply RepairsApi.Tests/V2/Controllers/GasServiceBoilersControllerTests.cs:162]: Not a problem, good to merge

**Pull request merged**